pages/support.vue

```diff
@@ -18,7 +20,7 @@
             </template>
             <template v-slot:team>
               <NuxtLink to="/team" class="text-nuxt-green underline">
-                NuxtJS team
+                NuxtJS ekibi
```

Comment:
changing this here will change this for all languages. If we need to translate the word team then we need to modify this by adding this phrase to the i18n file and then calling it here. obviously we then need to adjust all other languages. I can do this if you like and then you can just translate the word team when it is done. so for now you can remove all changes to this file.

Reply:
Wow, alright. Since it's not a big deal I'll just keep it default until you add those strings to i18n 🥰🙌